presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkLocalShuffleReadInfo.java

```diff
@@ -22,14 +22,14 @@
 public class PrestoSparkLocalShuffleReadInfo
         implements PrestoSparkShuffleReadInfo
 {
-    private final int maxBytesPerPartition;
+    private final long maxBytesPerPartition;
```

Comment:
FYI after some refactoring in C++ side, for local shuffle we only need numPartitions and rootPath for read info now, which is equivalent to write. Would you be able to help to refactor in this commit?